src/com/oltpbenchmark/benchmarks/ycsb/dialects/cassandra-dialects.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<dialects>
+    <dialect type="CASSANDRA">
+
+        <procedure name="ScanRecord">
+            <statement name="scanStmt">
+                SELECT * FROM USERTABLE WHERE YCSB_KEY &gt; ? AND YCSB_KEY &lt; ? ALLOW FILTERING
+            </statement>
+        </procedure>
+        <procedure name="ReadModifyWriteRecord">
+            <statement name="selectStmt">
+                Select * from USERTABLE where YCSB_KEY=? 

```

Comment:
Do you really need this? Please use consistent captialization

Reply:
I think we do not need this. Removed.